src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -55,15 +59,24 @@ public ArrayList<Integer> getFavouriteNumbers() {
          Use the ArrayList's replaceAll method to iterate through the ArrayList and replace each value with its double
          https://www.programiz.com/java-programming/library/arraylist/replaceall
      */
+    public ArrayList<Integer> multiply(ArrayList<Integer> list, int num){
+
 
+        list.replaceAll(e -> e*num);
+
+        return list;
+    }
 
 
     /*
         TODO: 3. Create a method named isEmpty that accepts one parameter:
          - A list of strings
          The method must return a boolean that indicates whether the provided list is empty or not
      */
-
+    public boolean isEmpty(ArrayList<String> list){
+        if (list.isEmpty())return true;
+        else return false;
+    }
```

Comment:
Can be simplified, eg:

public boolean isEmpty(ArrayList<String> list) {
    return list.isEmpty();
}